### DISCUSSION TITLE
Show more details during break

### DISCUSSION THREAD

**Discussion by CloneWith** [ppy/osu#27131] [Ideas]
During break of a gameplay, only accuracy and grade are shown under the countdown, which might be not enough.

can add those details and make them customizable:

- Performance points
- Unstable rate
- Progress (like the one on the pause screen)

**Comment by Rekunan** [comment]
I think it should only show details if you added a counter for them, but maybe there could be a customization option to choose what shows up, and maybe have some only show up in break instead of all the time.

**Comment by CloneWith** [comment]
> maybe there could be a customization option to choose what shows up, and maybe have some only show up in break instead of all the time.

yep, this makes it possible to imitate the classic behaviour of osu!stable, which might be familiar to most players
